seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/utils/PeekBlockingQueue.java

```diff
@@ -45,10 +48,19 @@ public class PeekBlockingQueue<E> {
     private final Lock lock = new ReentrantLock();
     private final Condition notEmpty = lock.newCondition();
 
+    private final Map<Long, E> jobIdMap = new ConcurrentHashMap<>();
+    private final Function<E, Long> jobIdExtractor;
```

Comment:
```suggestion
    private final Function<E, Long> idExtractor;
```
Change jobid to id, so that we can use this tool in other place.

Reply:
updated